.gitignore

```diff
@@ -10,7 +10,7 @@ tools/dind/config/
 tools/dind/config.json*
 tools/dind/apps.json
 tools/dind/backup.tgz
-test/data/config*.json
+test/data/**/config*.json
```

Comment:
What's this addition for ?

Reply:
The new test creates a configfs.json in `test/data/mnt/boot` which we don't want in the repo

Reply:
ah ok. I was just making sure since it had the wildcard match.